processor/processor.go

```diff
@@ -2832,16 +2865,94 @@ func (proc *Handle) userTransformAndFilter(
 	}
 	// Send to custom transformer only if the destination has a transformer enabled
 	if transformationEnabled {
-		userTransformationStat := proc.newUserTransformationStat(sourceID, workspaceID, destination)
-		userTransformationStat.numEvents.Count(len(eventList))
-		proc.logger.Debug("Custom Transform input size", len(eventList))
+		noOfEvents := len(eventList)
+		utMirroringEnabled, utMirroringSanityChecks := proc.isUserTransformMirroringEnabled()
```

Comment:
Do you think we should make this configurable at transformation level? current code enables for whole namespace, and there might be transformations that have external API calls and because of that, we can't test the others as well. 
Other way to do is have a configurable filter for transformations for which we don't want to do mirror test.

Reply:
Given that the main goal is to measure the performance gain, once we enabled mirroring we'd need to enable it for the whole namespace so that we can compare results. We'll have to enable this only for selected customers and for brief periods of time while we got it under control (e.g. after we review the transformations and make sure no external calls are made etc...).